provisioner/ibm-s3fs-provisioner.go

```diff
@@ -357,6 +365,20 @@ func (p *IBMS3fsProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 		sess = p.Backend.NewObjectStorageSession(sc.OSEndpoint, sc.OSStorageClass, creds, p.Logger)
 	}
 
+	if len(allowedNamespace) > 0 {
+		var allowed = false
+
+		for _, item := range allowedNamespace {
+			if item == pvcNamespace {
+				allowed = true
+				break
+			}
+		}
+		if allowed == false {
+			return nil, errors.New(pvcName + ":" + clusterID + ":cannot create bucket as PVC creation in this namespace is not allowed")
```

Comment:
Here in this code section, we are not checking whether `auto-create-bucket: true` or `auto-create-bucket: false`, so I think we should replace error msg `cannot create bucket as PVC creation in this namespace is not allowed` -----> ` PVC creation in 
 '<pvc_namespace_name>' namespace is not allowed`